setup.py

```diff
@@ -95,19 +78,21 @@ def compile_cuda_module(host_args):
     _, nvcc = get_cuda_path()
 
     print("NVCC ==> ", nvcc)
-    arch_flags = "-arch=sm_37 " + \
-                 "-gencode=arch=compute_37,code=sm_37 " + \
-                 "-gencode=arch=compute_70,code=sm_70 " + \
-                 "-gencode=arch=compute_75,code=sm_75 " + \
-                 "-gencode=arch=compute_75,code=compute_75"
-    nvcc_command = "-allow-unsupported-compiler shap/cext/_cext_gpu.cu -lib -o {} -Xcompiler {} -I{} " \
```

Comment:
As part of this PR, I noticed there is a space missing between the `-I / --include-path` flag and the value.